OAuth2TV/ViewController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ViewController.swift
+//  OAuth2TV
+//
+//  Created by Martin on 16/07/2018.
+//  Copyright © 2018 Auth0. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
what's the point on keeping this class if it only overrides 2 methods that call the super? as opposed to  `OAuth2Mac/ViewController.swift` which does change one of the methods logic.

Reply:
Again this is Apple's boilerplate application creation.